packages/node/buda.proto

```diff
@@ -146,6 +146,11 @@ message Visit {
 		string content = 4;
 		string term = 5;
 	}
+	message Experiment {
```

Comment:
This change seems the same as in the file `buda/entities/visit.proto`. Curious if we could replace it with a link to avoid copy pasting code here.

Reply:
@davidgasquez yep, @djfarrelly is correct. I've actually been debating weither or not I should perhaps add this file to the `.gitignore`, not sure if that's a good idea or not?